externs/browser/w3c_geometry1.js

```diff
@@ -342,3 +342,820 @@ DOMPointInit.prototype.z;
  * @suppress {duplicate}
  */
 DOMPointInit.prototype.w;
+
+/**
+ * @constructor
+ * @param {string|Array<number>} init
```

Comment:
This should have been marked as optional according to the spec at https://www.w3.org/TR/geometry-1/#dommatrix.

![dom_matrix_constructor](https://user-images.githubusercontent.com/7547039/107207641-633ef700-69b5-11eb-9688-c1a272b0d6bd.png)

I'll file a separate bug to track this.